ActivityAnalyses/LambdaFunctions/analyzeGait.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Jul  5 13:23:12 2023
+
+this function performs kinematic gait analysis
+
+@authors: Scott Uhlrich & Antoine Falisse
+"""
+
+import os
+import sys
+sys.path.append("../../")
+sys.path.append("../ActivityClasses")
+
+from gaitAnalysis import gait_analysis
+import utils
+
+def analyzeGait(trial_id,scalar_names=None,n_gait_cycles=1,filterFreq=6):
```

Comment:
I would call this an example instead and move it to the Examples folder. I think it is better to have all the examples in one place. EG, `example_gait_metrics`. It would be nice to add some plots to the example, for instance of mean +/- std of joint angles and some print outs of metrics, for instance the gait speed was ...